custom/v3/commands/data/src/main/java/org/apache/fineract/v3/commands/data/TypedCommand.java

```diff
@@ -57,7 +47,7 @@ public JsonCommand getJsonCommand() {
         }
 
         return new JsonCommand(null, json, null, null, entityName, resourceId,
```

Comment:
We have a "CommandWrapper" and a "JsonCommand"... why? Can't we just express what we need in one class?

Reply:
Yes, I've thought about that, there's no major differences. (just action & entity names).
Thanks for mentioning it!